### ISSUE TITLE
wire.proto not in repo?

### ISSUE THREAD

**Issue by nornagon** [opentracing/basictracer-python#19]
It looks like the generated file is in the repo, but not the Proto source?

**Comment by bensigelman** [comment]
@nornagon that's correct (I'm not suggesting it's better this way!). The file can be found here: https://github.com/opentracing/basictracer-go/blob/master/wire/wire.proto

The "right" (or "righter") thing would be to factor out the protos into something like a `basictracer-common` repo and add said repo as a submodule here and in other basictracer repos. If you want to take this on, I can help with the mechanics.

**Comment by nornagon** [comment]
Ah, I see, that makes sense. For [xray-python-opentracing](https://github.com/nornagon/xray-python-opentracing) I ended up rewriting the binary format to not depend on protobuf (since I needed to serialize trace ids as strings, and the format is pretty simple). I get that pb makes more sense for language interop though.

I'd be happy to take on that refactor. The only bit I wouldn't be able to make a PR for would be creating the new repo.

**Comment by bhs** [comment]
@nornagon I think it's also permissible to ignore the binary format while you work out the kinks of the rest of the lib... IIRC the basictracer-python stuff even has some weird code to avoid the protobuf imports entirely since they caused some packaging issues in production codebases.
